packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -126,7 +130,7 @@ async function handleUrlSourceAsync(
   }
 
   return {
-    url,
+    resolvedArchiveSource: { type: SubmissionArchiveSourceType.Url, url },
     source,
```

Comment:
Couldn't this be sth like:
```suggestion
    source: { type: SubmissionArchiveSourceType.Url, url },
```